pilot/pkg/networking/apigen/apigen_test.go

```diff
@@ -38,7 +38,7 @@ var (
 
 // Creates an in-process discovery server, using the same code as Istiod, but
 // backed by an in-memory config and endpoint store.
-func initDS() *xds.Server {
+func initDS() *xds.SimpleServer {
```

Comment:
This was changed to `SimpleServer` because not `DiscoveryServer` is also in xds package and can be confusing to have both.